### DISCUSSION TITLE
No effect for calling "nsys launch"

### DISCUSSION THREAD

**Discussion by yuwenchen95** [JuliaGPU/CUDA.jl#1939] [Q&A]
I am trying to profile my code. It works fine if I run
`nsys profile C:\Users\ddt00\AppData\Local\Programs\julia-1.8.1\bin\julia-1.8.1.exe`

However, when I was running
`nsys launch C:\Users\ddt00\AppData\Local\Programs\julia-1.8.1\bin\julia-1.8.1.exe`
it returns
`command ignored` without any error and nothing happened.

Has anyone encountered such an issue before?

**Comment by maleadt** [comment]
Let's make this an issue, as it isn't really a bug with CUDA.jl you're reporting here.

I have encountered this before when there were other nsys-related processes running, so try looking for those and killing them first.

**Reply by maleadt** [reply]
> try looking for those and killing them first.

I'm going to assume that helped.
